app/models/interest.rb

```diff
@@ -0,0 +1,3 @@
+class Interest < ApplicationRecord
+  belongs_to :user
```

Comment:
I think it isn't clear but looking at #5 I think a user can have many interests.

I will ask project creator about clarifying it.